pgvectorscale/src/access_method/mod.rs

```diff
@@ -99,16 +102,30 @@ BEGIN
     AND c.opcmethod = (SELECT oid FROM pg_catalog.pg_am am  WHERE am.amname = 'diskann');
 
     IF c = 0 THEN
-        CREATE OPERATOR CLASS vector_cosine_ops DEFAULT
+        CREATE OPERATOR CLASS vector_cosine_ops
```

Comment:
I think this needs to be a bit more involved. There is now a chance that vector_cosine_ops exists but without the Function 1. In that case, we need to drop the old ops and recreate. We should try to keep this logic idempotent.

Reply:
I would also investigate why the tests are passing. The upgrade test should fail now. Is it not being run?

Reply:
@cevian The upgrade test [is `#[ignore]`d](https://github.com/timescale/pgvectorscale/blob/9802ac91db55cfea05eb10b717b13fe736eb101c/pgvectorscale/src/access_method/upgrade_test.rs#L25-L27), and only run manually